ocaml/xapi-aux/throttle.mli

```diff
@@ -34,21 +34,27 @@ module Batching : sig
    *)
 
   val with_recursive_loop : t -> (('a -> 'b) -> 'a -> 'b) -> 'a -> 'b
-  (** [with_recursive_loop config f arg] calls [f self arg], where [self] can be used
+  (** [with_recursive config f arg] calls [f self arg], where [self] can be used
```

Comment:
```suggestion
 (** [with_recursive_loop config f arg] calls [f self arg], where [self] can be used
```

Reply:
This got accidentally reverted in 'Use exponential backoff', fixed up that commit.